source/grid/grid_reordering.cc

```diff
@@ -251,6 +269,19 @@ namespace internal
     };
 
 
+
+    /**
+     * A class that represents all of the cells adjacent to a given edge.
+     * This class corresponds to the 3d case where each edge can have an
+     * arbitrary number of adjacent cells. We represent this as a
+     * std::set<AdjacentCell>, from which class the current one is
```

Comment:
typo: 'we represent this as a `std::set`' should be 'we represent this as a `std::vector`